### PR TITLE
Providing explicit documentation for throwing an exception when no bu…

### PR DESCRIPTION
**Pull request by cjdb**
…ffer exists.

### CODE REVIEW

File: placeholder_accessors/index.md [hunk 1]
```diff
@@ -199,8 +199,29 @@ Another method, that allows specifying the memory object the placeholder
 accessor will be associated is also provided:
 
 `handler::require(buffer<T, dim> b,
-    accessor<T, dim, mode, target, acess::placeholder::true_t>)`
+    accessor<T, dim, mode, target, access::placeholder::true_t>)`
 
+### `accessor`s without buffers
```
Comment (AerialMantis) [inline placeholder_accessors/index.md]: `accessor`s without buffers -> Placeholder `accessor` without a buffer.

File: placeholder_accessors/index.md [hunk 2]
```diff
@@ -199,8 +199,29 @@ Another method, that allows specifying the memory object the placeholder
...
+
+If a placeholder accessor is not tied to a buffer within a command group, then
```
Comment (AerialMantis) [inline placeholder_accessors/index.md]: `If a placeholder accessor` -> `If a placeholder accessor which was not constructed with a buffer`.

File: placeholder_accessors/index.md [hunk 3]
```diff
@@ -201,27 +201,16 @@ accessor will be associated is also provided:
...
     accessor<T, dim, mode, target, access::placeholder::true_t>)`
 
-### `accessor`s without buffers
+### Placeholder `accessor` without a buffer
 
-If a placeholder accessor is not tied to a buffer within a command group, then
-an exception is thrown.
+If a placeholder accessor which was not constructed with a buffer is not tied
+to a buffer within a command group, then an exception is thrown. An accessor
+can be checked for a buffer using `has_buffer()`.
```
Comment (Ruyk) [inline placeholder_accessors/index.md]: for _the existence of an associated_ buffer

File: placeholder_accessors/index.md [hunk 4]
```diff
@@ -201,27 +201,16 @@ accessor will be associated is also provided:
...
-      accessor<T, dim, mode, target, access::placeholder::true_t> a)
-{
-   if (not a.has_buffer())
-      throw std::runtime_error{"No buffer in accessor."};
-   // ...
-}
-```
+|Member function  |Description                                              |
+|-----------------|---------------------------------------------------------|
+|bool has_buffer()|Returns true if the accessor is associated with a buffer,|
```
Comment (Ruyk) [inline placeholder_accessors/index.md]: ```bool has_buffer() const``` to prevent side effects

Comment (cjdb) [inline placeholder_accessors/index.md]: Oops!

**Comment by cjdb** [comment]
@Ruyk for your review

**Pull request merged**